qemu2deb.sh

```diff
@@ -262,7 +262,7 @@ function intro() {
     #added extra 5 "spaces" for '-beta'
     echo -e "
     ###########################################
-    #  QEMU2DEB $APPVER by Itai-Nelken | 2021   #
```

Comment:
The spaces here are intentional as when `$APPVER` is expanded, it uses less space.